[32mINFO    [0m integration.test_data_integrator:test_data_integrator.py:102 Ensuring no data exists in the test database
[32mINFO    [0m integration.helpers:helpers.py:468 Running get-credentials on data-integrator/0
[32mINFO    [0m integration.test_data_integrator:test_data_integrator.py:130 Relating mysqlrouter with self-signed-certificates
[33mWARNING [0m juju.model:model.py:1666 relate is deprecated and will be removed. Use integrate instead.
[32mINFO    [0m integration.test_data_integrator:test_data_integrator.py:176 Removing relation between mysqlrouter and self-signed-certificates
[32mINFO    [0m pytest_operator.plugin:plugin.py:862 Model status:

Model    Controller     Cloud/Region         Version  SLA          Timestamp
testing  concierge-lxd  localhost/localhost  3.6.2    unsupported  01:34:55Z

App                       Version          Status  Scale  Charm                     Channel        Rev  Exposed  Message
data-integrator                            active      1  data-integrator           latest/stable   78  no       
mysql                     8.0.41-0ubun...  active      1  mysql                     8.0/edge       349  no       
mysql-router              8.0.41-0ubun...  active      1  mysql-router                               0  no       
self-signed-certificates                   active      1  self-signed-certificates  latest/stable  155  no       

Unit                         Workload  Agent  Machine  Public address  Ports           Message
data-integrator/0*           active    idle   1        10.251.162.75                   
  mysql-router/0*            active    idle            10.251.162.75   6446-6447/tcp   
mysql/0*                     active    idle   0        10.251.162.40   3306,33060/tcp  Primary
self-signed-certificates/0*  active    idle   2        10.251.162.202                  

Machine  State    Address         Inst id        Base          AZ  Message
0        started  10.251.162.40   juju-05af83-0  ubuntu@22.04      Running
1        started  10.251.162.75   juju-05af83-1  ubuntu@22.04      Running
2        started  10.251.162.202  juju-05af83-2  ubuntu@22.04      Running

[32mINFO    [0m pytest_operator.plugin:plugin.py:868 Juju error logs:

machine-0: 01:31:32 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-0: 01:31:32 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
unit-mysql-0: 01:31:32 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
machine-2: 01:33:08 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-2: 01:33:08 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
unit-self-signed-certificates-0: 01:33:08 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
machine-1: 01:33:10 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-1: 01:33:10 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
unit-data-integrator-0: 01:33:10 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
unit-mysql-router-0: 01:33:15 ERROR juju.worker.meterstatus error running "meter-status-changed": exit status 1
unit-self-signed-certificates-0: 01:34:54 ERROR juju.worker.dependency "uniter" manifold worker returned unexpected error: relation scope id "ae82d97e-c7d1-4ba8-81ce-87ac4e05af83:r#10#mysql-router": settings ae82d97e-c7d1-4ba8-81ce-87ac4e05af83:r#10#mysql-router not found (not found)

[32mINFO    [0m pytest_operator.plugin:plugin.py:947 Forgetting model main...